aiida/cmdline/commands/cmd_status.py

```diff
@@ -90,8 +103,9 @@ def verdi_status():
     # getting the rmq status
     try:
         with Capturing(capture_stderr=True):
-            comm = manager.create_communicator(with_orm=False)
-            comm.stop()
+            with set_log_level(logging.getLogger('kiwipy'), logging.CRITICAL):
```

Comment:
Looking at the original issue #2914 I wonder if this will be enough.
For one it seems to throw log messages in two checks:
 * postgres
 * rabbitmq

Apparently, for the code path of the first, it also hits the `self._communicator.connect()` which tries to establish a connection to the RabbitMQ and so that fails just as the RabbitMQ test itself fails. It would be good if we can figure out why the database test hits code that tries to connect to RabbitMQ and see if we can isolate it to have a more pure database-only test.

Secondly, the log messages do not just come from `kiwipy`, but also from `tornado` and `pika`, two other libraries that `kiwipy` depends on. Are we sure that all messages will now be surpressed? I know this is very difficult to put in a unit test, but did you test this locally?

Should we maybe update the `set_log_level` context manager to simply suppress all loggers for the time being? Or capture it and redirect it optionally if the `Exception` clause is triggered?

Reply:
> Looking at the original issue #2914 I wonder if this will be enough.
> For one it seems to throw log messages in two checks:
> 
> * postgres
> * rabbitmq
> 
> Apparently, for the code path of the first, it also hits the `self._communicator.connect()` which tries to establish a connection to the RabbitMQ and so that fails just as the RabbitMQ test itself fails. It would be good if we can figure out why the database test hits code that tries to connect to RabbitMQ and see if we can isolate it to have a more pure database-only test.

I'd argue that this is out-of-scope for this PR.

> Secondly, the log messages do not just come from `kiwipy`, but also from `tornado` and `pika`, two other libraries that `kiwipy` depends on. Are we sure that all messages will now be surpressed? I know this is very difficult to put in a unit test, but did you test this 

I've tested this locally. How are the command line entrypoints tested at the moment?

> Should we maybe update the `set_log_level` context manager to simply suppress all loggers for the time being? Or capture it and redirect it optionally if the `Exception` clause is triggered?

Agreed, which is why I've adjusted the patch as described in my comment above.

Reply:
I agree that removing the `self._communicator.connect()` path from the postgres connect pathway is out of the scope, however, it is resulting in error logs printed to screen in `verdi status` _does_ seem to be in scope. It is almost exactly the title of the issue. All I would suggest is to place the same temporary logging disabling around the attempt to connect to postgres as well. It is the try-catch block just above the one you are already targeting.

The `verdi` command tests are in `aiida.backends.tests.cmdline.commands`. They are run through `verdi -p <test_profile> devel tests db.cmdline.commands`.

Reply:
I've made an attempt at implementing a unit test for this issue and have so far not succeeded. Unless you know an elegant way to trigger the faulty behavior, I'd like to merge this as-is.

Reply:
That is fine, this would be tricky to test properly. If you just address the last two comments then we can merge